src/Runner.ts

```diff
@@ -75,7 +75,7 @@ export async function createRunner(
   if (!extension) {
     const { default: puppeteer } = await import('puppeteer');
     const browser = await puppeteer.launch({
-      headless: true,
+      headless: !process.env.PUPPETEER_HEADFUL,
```

Comment:
Thanks for the PR, but I think it's better to implement this in CLIUtils:runFiles https://github.com/puppeteer/replay/blob/main/src/CLIUtils.ts#L34

We cannot include the change in `src/Runner.ts` as it is used by DevTools which is a non-Node envrionment and does not have `process.env`.

Reply:
> We cannot include the change in `src/Runner.ts` as it is used by DevTools which is a non-Node envrionment and does not have `process.env`.

The problem I see with that, is that it would require changes at the invocation of `createRunner`, which at the moment, only passes `recording` as argument.

Thus, even if adding an option to the `opts` of the `runFiles` function, it would need a change in the call of `createRunner`, which would need a change in its signature because its second (optional) argument is `extension` (aka, it needs a third argument).  
And `createRunner` would also need some changes to its body.

I also considered creating my own extension (to pass to a call to `createRunner`)... That doesn't really solve the problem it would also require changes to `runFiles`.

Since I don't measure very well the possible side-effects of those changes, like you do.

I think I'll be faster writing my own little script to run the JSON files.

Reply:
I think we can land the following PR https://github.com/puppeteer/replay/pull/130 That should offer equivalent functionality